photon-ml/src/main/scala/com/linkedin/photon/ml/function/svm/SmoothedHingeLossLinearSVMFunction.scala

```diff
@@ -12,33 +12,19 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-package com.linkedin.photon.ml.function
+package com.linkedin.photon.ml.function.svm
```

Comment:
I'm surprised that package "svm" is inside package "function". I thought "svm" referred to "support vector machine", which is a whole model. Wikipedia for example says: "In machine learning, support vector machines are **supervised learning models** with associated learning algorithms that analyze data used for classification and regression analysis." So, they would not be "functions"?